e2e/terraform/configs/client.hcl

```diff
@@ -0,0 +1,20 @@
+data_dir = "/opt/nomad/data"
+bind_addr = "0.0.0.0"
+
+# Enable the client
+client {
+  enabled = true
+  options {
+    "driver.raw_exec.enable" = "1"
```

Comment:
These options will be deprecated in 0.9. We should switch to using the `plugin` block syntax.

Ex:
```
plugin "raw_exec" {
  config {
    enabled = true
  }
}

plugin "docker" {
  config {
    allow_privileged = true
  }
}
```